src/searching/searching.py

```diff
@@ -30,9 +34,13 @@ def binary_search(arr, target):
 
 # STRETCH: write a recursive implementation of Binary Search
 def binary_search_recursive(arr, target, low, high):
```

Comment:
Binary search recursive seems most intuitive when it comes to recursion as a concept